crates/biome_graphql_parser/src/parser/definitions/enum.rs

```diff
@@ -34,6 +34,28 @@ pub(crate) fn parse_enum_type_definition(p: &mut GraphqlParser) -> ParsedSyntax
     Present(m.complete(p, GRAPHQL_ENUM_TYPE_DEFINITION))
 }
 
+/// Must only be called if the next 2 token is `extend` and `enum`, otherwise it will panic.
+#[inline]
+pub(crate) fn parse_enum_type_extension(p: &mut GraphqlParser) -> ParsedSyntax {
+    let m = p.start();
+
+    p.bump(T![extend]);
+    p.bump(T![enum]);
```

Comment:
```suggestion
    p.expect(T![enum]);
```

Reply:
I don't know if this is truly necessary, as this parsing rule should only be called in one place ([here](https://github.com/biomejs/biome/blob/d1de19cb9c8751444f58e35563544c2659b2151d/crates/biome_graphql_parser/src/parser/definitions/mod.rs#L104)), and we must have already checked for the presence of both tokens before advancing. Correct me if you think otherwise.

Reply:
Yeah, it’s not a blocker, but it’s good practice for the parser functions to validate their preconditions instead of assuming to be called correctly. It wouldn’t change anything now, but it brings some extra robustness in the face of future changes to the parser.